libnmstate/netapplier.py

```diff
@@ -315,7 +318,35 @@ def assert_ifaces_state(ifaces_desired_state, ifaces_current_state):
 
         if iface_dstate != iface_cstate:
             raise DesiredStateIsNotCurrentError(
-                'desired: {}\ncurrent: {}'.format(iface_dstate, iface_cstate))
+                _format_desired_current_state_diff(iface_dstate,
+                                                   iface_cstate)
+            )
+
+
+def _format_desired_current_state_diff(desired_state, current_state):
+    pretty_desired_state = PrettyState(desired_state).yaml
+    pretty_current_state = PrettyState(current_state).yaml
+
+    diff = "".join(difflib.unified_diff(
```

Comment:
Single quotes please (unlit line 334)

Reply:
fixed... How about we switch to double quotes? Then we can use the black format style as well, add it to tox and do not have to change this by hand. :-) https://github.com/ambv/black

Or is there some other automatic code formatter that we could use that defaults to single ticks?